substra/sdk/client.py

```diff
@@ -53,6 +53,26 @@ def get_asset_key(data):
     return data.get('pkhash') or data.get('key')
 
 
+def _flatten_metadata(data, field_name):
+    data = deepcopy(data)
+
+    for i in data[field_name]:
+        data[f'{field_name}_{i}'] = data[field_name].get(i, [])
```

Comment:
Since you're already iterating on the keys of `data[field_name]`, you are sure that data[field_name][i] is defined, you don't need to use `.get()`.